src/jvm/main/org/jetbrains/kotlinx/lincheck/UnsafeHolder.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Lincheck
+ *
+ * Copyright (C) 2019 - 2024 JetBrains s.r.o.
+ *
+ * This Source Code Form is subject to the terms of the
+ * Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed
+ * with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.jetbrains.kotlinx.lincheck
+
+import sun.misc.*
+
+object UnsafeHolder {
```

Comment:
Is it used only for byte-code transformation?
Can we move it into transformation sub-package if so?

Reply:
Moved